docs/connect/cli.md

```diff
@@ -122,6 +122,8 @@ option and the URI pattern `redis://user:password@host:port/dbnum`:
     $ redis-cli -u redis://LJenkins:p%40ssw0rd@redis-16379.hosted.com:16379/0 PING
     PONG
 
+**NOTE:** User, password and dbnum are optional. For authentication without a username, use username `default`. For TLS, use the scheme `rediss`.
```

Comment:
I will put one sentence per line. It is not a visible linebreak on the page, only in the markdown code. This is according to the guideline for the doc repo: https://github.com/redis/redis-doc#styling-guidelines

```suggestion
**NOTE:**
User, password and dbnum are optional.
For authentication without a username, use username `default`.
For TLS, use the scheme `rediss`.
```